### PR TITLE
IOTA-313 add CK editor to the field

### PR DESCRIPTION
**Pull request by vesnacustic**
CK editor added as a requirement in `package.json`, and webpack config is added to build the js file that loads and sets up the CK editor on the `nghtmltext` field in backoffice. CK editor is still not available in adminui.

Run `yarn encore dev --config=webpack.config.js` to build JS.

To test, move to [IOTA-313-nghtmltext-field-type](https://bitbucket.org/netgen/iotasite/branch/IOTA-313-nghtmltext-field-type) branch on iota project and run `composer install` there as `netgen/ibexa-fieldtype-html-text` bundle is added as a dependency. 
You should then be able to add HTML text field to a content type - to test, please use country content type as mock controller has been added for country edit. 

Then go to `/iota/admin/country/edit/{contentId}/{versionNo}/{languageCode}/{locationId}` route with an existing Content (version should be draft!) and try to edit HTML text field using CK editor. Check that your modifications are saved in the content you edited.

NOTE: just realized field validation is missing, will work on that next.

### CODE REVIEW

File: bundle/Resources/js/app.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+
+import './basePath.js';
```
Comment (emodric) [inline bundle/Resources/js/app.js]: If there's no technical reason why this is separated to a different file, I'd just include it here.

File: bundle/Form/Field/NgHtmlTextValueTransformer.php [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\IbexaFieldTypeHtmlTextBundle\Form\Field;
+
+use Ibexa\AdminUi\Form\DataTransformer\FieldType\FieldValueTransformer;
+
+class NgHtmlTextValueTransformer extends FieldValueTransformer
```
Comment (emodric) [inline bundle/Form/Field/NgHtmlTextValueTransformer.php:9]: Why do we need an empty transformer? For validation you mentioned that is missing or?

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    .setPublicPath(`/bundles/netgenibexafieldtypehtmltext/build`)
+
+    .addEntry('app', `./bundle/Resources/js/app.js`)
+
+    .setManifestKeyPrefix('build/')
+
+    // empty the outputPath dir before each build
+    .cleanupOutputBeforeBuild()
+    .copyFiles([
+        { from: './node_modules/ckeditor4/', to: 'ckeditor/[path][name].[ext]', pattern: /\.(js|css)$/, includeSubdirectories: false },
```
Comment (emodric) [inline webpack.config.js]: Why not use ckeditor 5?

File: bundle/Resources/js/app.js (hunk 1 above)
Comment (vesnacustic) [reply bundle/Resources/js/app.js]: As ckeditor is now used as a separate build, this import is not needed so file is deleted in https://github.com/netgen/ibexa-fieldtype-html-text/pull/3/commits/c93c56cfb6716a82db7e6f5b25d923edcc68027e

File: bundle/Form/Field/NgHtmlTextValueTransformer.php (hunk 2 above)
Comment (vesnacustic) [reply bundle/Form/Field/NgHtmlTextValueTransformer.php:9]: As we discussed in person, this file will be edited in the next commit (when adding validation) so we'll keep it here for now.

File: webpack.config.js (hunk 3 above)
Comment (vesnacustic) [reply webpack.config.js]: ckeditor5 now used, changed in https://github.com/netgen/ibexa-fieldtype-html-text/pull/3/commits/c93c56cfb6716a82db7e6f5b25d923edcc68027e

**Comment by emodric** [comment]
Thank you @vesnacustic !

**Pull request merged**